news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Performance Points & Star Rating Updates
+date: 2024-09-17 12:00 +0000
+---
+
+The time for change is finally here!
+
+![](/wiki/shared/news/banners/pp-sr-osu.jpg)
+
+There are changes to all game modes this time around in an effort to improve the difficulty and performance calculations. This news post will discuss what has changed in a format that should be understandable to you, the player!
+
+If you find yourself scratching your head when reading, consider consulting the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) wiki articles to gain a greater understanding of the topics.
+
+## Release schedule
+
+<!-- todo -->
+
+---
+
+## osu!
+
+All changes for osu! are covered in the osu!(lazer) update video, which you may be interested in watching:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Kqu4TUOO5IY" frameborder="0" allowfullscreen></iframe>
+</div>
+
+As a reminder of the various core values in the difficulty calculation algorithm:
+
+- **Aim**: how difficult it is to consistently hit notes back-to-back (affected by approach rate, jumps, Hidden, Hard Rock and more)
+- **Speed**: how quickly new elements are presented to the player (affected by note density, BPM, Double Time and Half Time)
+- **Accuracy**: how difficult it is to stay rhythmically on time (affected by Hidden, Hard Rock, Flashlight and more)
+- **Strain**: the difficulty of enduring intense or technically demanding patterns often and for a long time (affected by streams, fast jumps and more)
+
+### Combo scaling removal
+
+The most monumental [change](https://github.com/ppy/osu/pull/16280) proposed by [moonpoint](https://osu.ppy.sh/users/9558549) – with improvements made by [KermitNuggies](https://osu.ppy.sh/users/33452559) – removes the "combo scaling factor" applied to the aim and speed portions of a score's PP.
```

Comment:
"pp" should probably be lowercase as its written like that on the performance points wiki page and i dont think the community uses it in uppercase
```suggestion
The most monumental [change](https://github.com/ppy/osu/pull/16280) proposed by [moonpoint](https://osu.ppy.sh/users/9558549) – with improvements made by [KermitNuggies](https://osu.ppy.sh/users/33452559) – removes the "combo scaling factor" applied to the aim and speed portions of a score's pp.
```